Bottom: 55ffa8849cf2c6e18856f1e9d278573786bcb7f0
Top:    efcbaa1f674e8166f36fe2274c9c14e195a30159
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-04-16 22:01:51 -0700

rsocket: Remove requirement for minimal inline data

Inline data support is optional.  Allow rsockets to work
with devices that do not support inline data, such as the
Intel TrueScale HCA.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/rsocket.c b/src/rsocket.c
index 8325edf..7c5083c 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -860,7 +860,7 @@ static int rs_create_ep(struct rsocket *rs)
 	if (ret)
 		return ret;
 
-	rs->sq_inline = qp_attr->cap.max_inline_size;
+	rs->sq_inline = qp_attr.cap.max_inline_data;
 	if (rs->sq_inline < RS_MIN_INLINE)
 		return ERR(EINVAL);
